src/components/clusters/wizards/form/FileUploadField.tsx

```diff
@@ -74,7 +74,7 @@ export const FileUploadField = ({
           <FormGroup
             fieldId={field.name}
             label={label}
-            {...(tooltip && { labelIcon: <PopoverHint hint={tooltip} /> })}
+            {...(tooltip && { labelHelp: <PopoverHint hint={tooltip} /> })}
```

Comment:
I was asked to clean up error messages in my unit tests,  This incorrect usage of labelIcon (prop does not exist in PF6) was throwing an error when running my tests.  I have fixed it.